pytorch_lightning/core/lightning.py

```diff
@@ -942,28 +942,36 @@ def configure_optimizers(self):
                     dis_sched = CosineAnnealing(discriminator_opt, T_max=10) # called every epoch
                     return [gen_opt, dis_opt], [gen_sched, dis_sched]
 
-        Some things to know
+        Some things to note:
 
-            - Lightning calls ``.backward()`` and ``.step()`` on each optimizer
```

Comment:
why is this listing removed?

Reply:
This section used a functionally different formatting in 0.6.0. It rendered properly.

<img width="760" alt="060" src="https://user-images.githubusercontent.com/1803297/76966369-da7a2100-68f3-11ea-80ea-ad62ac4709bd.png">

I noticed it was [changed here](https://github.com/PyTorchLightning/pytorch-lightning/commit/f996c2892f60b428b1c5ba4b397fa361103e5c65#diff-2d943a0d0806233840a33725d8a7ce37), which is its current state today =(.

<img width="616" alt="07x" src="https://user-images.githubusercontent.com/1803297/76966392-e4038900-68f3-11ea-9762-d8b622c5abec.png">

Attempting to understand the intent of the above commit, I believe it was recognized that `.. note::` was being abused and that no other section in the docs looked similar; hence it was removed completely.

I considered the option of bring back `.. note:`, back but under a single consolidated note, which would have rendered properly and is actually my desired outcome, very crisp and clean:

<img width="761" alt="060b" src="https://user-images.githubusercontent.com/1803297/76966407-e960d380-68f3-11ea-82f4-e108d9eba3b8.png">

But my commit adds a code block into this note section, and even without it, the note block is already sufficiently large. It really isn't a note but just part of the body/content of the section. I believe the formatting proposed in my current commit works the best.

---

Separately, in the current master / nightly, the entire trainer docs section is missing / isn't built (!) https://pytorch-lightning.readthedocs.io/en/latest/trainer.html due to `WARNING: autodoc: failed to import module ....` while doing a `make html`. However the cause of that is external to this PR.

Reply:
Well to be casted to the same item the new lines has to have one extra space at its beginning...
```
- first line starts 
 and continue here 
```